public/app/features/alerting/unified/components/silences/SilencesTable.tsx

```diff
@@ -22,40 +22,49 @@ const SilencesTable: FC<Props> = ({ silences, alertManagerAlerts, alertManagerSo
   };
   if (!!silences.length) {
     return (
-      <table className={tableStyles.table}>
-        <colgroup>
-          <col className={tableStyles.colExpand} />
-          <col className={styles.colState} />
-          <col className={styles.colMatchers} />
-          <col />
-          <col />
-          <col />
-        </colgroup>
-        <thead>
-          <tr>
-            <th />
-            <th>State</th>
-            <th>Matchers</th>
-            <th>Alerts</th>
-            <th>Schedule</th>
-            <th>Action</th>
-          </tr>
-        </thead>
-        <tbody>
-          {silences.map((silence, index) => {
-            const silencedAlerts = findSilencedAlerts(silence.id);
-            return (
-              <SilenceTableRow
-                key={silence.id}
-                silence={silence}
-                className={index % 2 === 0 ? tableStyles.evenRow : undefined}
-                silencedAlerts={silencedAlerts}
-                alertManagerSourceName={alertManagerSourceName}
-              />
-            );
-          })}
-        </tbody>
-      </table>
+      <>
+        <table className={tableStyles.table}>
+          <colgroup>
+            <col className={tableStyles.colExpand} />
+            <col className={styles.colState} />
+            <col className={styles.colMatchers} />
+            <col />
+            <col />
+            <col />
+          </colgroup>
+          <thead>
+            <tr>
+              <th />
+              <th>State</th>
+              <th>Matchers</th>
+              <th>Alerts</th>
+              <th>Schedule</th>
+              <th>Action</th>
+            </tr>
+          </thead>
+          <tbody>
+            {silences.map((silence, index) => {
+              const silencedAlerts = findSilencedAlerts(silence.id);
+              return (
+                <SilenceTableRow
+                  key={silence.id}
+                  silence={silence}
+                  className={index % 2 === 0 ? tableStyles.evenRow : undefined}
+                  silencedAlerts={silencedAlerts}
+                  alertManagerSourceName={alertManagerSourceName}
+                />
+              );
+            })}
+          </tbody>
+        </table>
+        <div className={styles.callout}>
+          <Icon className={styles.calloutIcon} name="info-circle" />
+          <span>
+            While you can’t delete a silence, a garbage collector automatically removes all expired-silences every 5
```

Comment:
Just a suggestion, but it'd keep this simple and not mention implementation details
```suggestion
          Expired silences are automatically deleted after 5 days.
```